config/settings/base.py

```diff
@@ -319,7 +310,7 @@
 
 # API Client
 
-CLIENT_BASE_URL = os.getenv("KONG_CLIENT_BASE_URL")
+CLIENT_BASE_URL = os.getenv("ROSETTA_DEV_CLIENT_BASE_URL")
```

Comment:
I think `CLIENT_BASE_URL` and `ROSETTA_DEV_CLIENT_BASE_URL` both need to be `ROSETTA_API_URL`